src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -821,39 +835,39 @@ def __init__(self,
         self.container_image_name = container_image_name  # image friendly name
         self.container_image_digests = container_image_digests  # reg hashes
 
-        # The type of service (osd, mon, mgr, etc.)
+        #: The type of service (osd, mon, mgr, etc.)
         self.daemon_type = daemon_type
 
-        # The orchestrator will have picked some names for daemons,
-        # typically either based on hostnames or on pod names.
-        # This is the <foo> in mds.<foo>, the ID that will appear
-        # in the FSMap/ServiceMap.
+        #: The orchestrator will have picked some names for daemons,
+        #: typically either based on hostnames or on pod names.
+        #: This is the <foo> in mds.<foo>, the ID that will appear
+        #: in the FSMap/ServiceMap.
         self.daemon_id: Optional[str] = daemon_id
 
-        # Some daemon types have a numeric rank assigned
+        #: Some daemon types have a numeric rank assigned
         self.rank: Optional[int] = rank
         self.rank_generation: Optional[int] = rank_generation
 
         self._service_name: Optional[str] = service_name
 
-        # Service version that was deployed
+        #: Service version that was deployed
         self.version = version
 
         # Service status: -1 error, 0 stopped, 1 running
```

Comment:
```suggestion
        # Service status: -2 unknown, -1 error, 0 stopped, 1 running, 2 starting
```
To match what you changed it to